packages/live-preview-sdk/src/inspectorMode/index.ts

```diff
@@ -239,10 +235,11 @@ export class InspectorMode {
       {
         elements: this.taggedElements.map((taggedElement) => ({
           // Important: do not add `element` as it can't be cloned by sendMessage
-          coordinates: taggedElement.coordinates,
-          isVisible: taggedElement.isVisible,
+          coordinates: taggedElement.coordinates!,
+          isVisible: !!taggedElement.isVisible,
```

Comment:
Could we fix the typings when we create the taggedElements?

Reply:
I've added a new type now

PrecaulculatedTaggedElement 

This just has the stuff that you can get directly from the tagged element, so element + attributes

The the actual TaggedElement has the calculated state without any conditional properties